monai/bundle/scripts.py

```diff
@@ -116,6 +121,212 @@ def _get_fake_spatial_shape(shape: Sequence[Union[str, int]], p: int = 1, n: int
     return tuple(ret)
 
 
+def _get_git_release_url(
+    repo_owner: str,
+    repo_name: str,
+    tag_name: str,
+    filename: str,
+):
+    return f"https://github.com/{repo_owner}/{repo_name}/releases/download/{tag_name}/{filename}"
+
+
+def _get_git_release_assets(
+    repo_owner: str,
+    repo_name: str,
+    tag_name: str,
+):
+    url = f"https://api.github.com/repos/{repo_owner}/{repo_name}/releases/tags/{tag_name}"
+    resp = requests_get(url)
+    resp.raise_for_status()
+    assets_list = json.loads(resp.text)["assets"]
+    assets_info = {}
+    for asset in assets_list:
+        assets_info[asset["name"]] = asset["browser_download_url"]
+    return assets_info
+
+
+def _download_from_github(
+    repo: str,
+    tag_name: str,
+    download_path: Path,
+    filename: Optional[str] = None,
+    progress: bool = True,
+    extract: bool = False,
+):
+    if len(repo.split("/")) != 2:
+        raise ValueError("if source is `github`, repo should be in the form of `repo_owner/repo_name`.")
+    repo_owner, repo_name = repo.split("/")
+    if filename is None:
+        # download the whole bundle package if filename is not provided
+        assets_info = _get_git_release_assets(repo_owner, repo_name, tag_name=tag_name)
+        for name, url in assets_info.items():
+            download_url(url=url, filepath=download_path / f"{name}", hash_val=None, progress=progress)
+        if extract is True:
+            logger.info("When download a whole bundle package, extract is not supported, skip extracting.")
+        logger.info(f"All files within the bundle package {tag_name} are downloaded.")
+    else:
+        # download a single file
+        url = _get_git_release_url(repo_owner, repo_name, tag_name=tag_name, filename=filename)
+        filepath = download_path / f"{filename}"
+        download_url(url=url, filepath=filepath, hash_val=None, progress=progress)
+        if extract is True:
+            extractall(filepath=filepath, output_dir=download_path, has_base=True)
+
+
+def download(
+    package: Optional[str] = None,
+    bundle_dir: Optional[PathLike] = None,
```

Comment:
I would suggest to use `filepath` as the arg directly, same as:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/apps/utils.py#L150

Thanks.